manifests/c/compuphase/pawn/4.1.7152/compuphase.pawn.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: compuphase.pawn
```

```suggestion
PackageIdentifier: Compuphase.Pawn
```
The validation is failing because the publisher `Compuphase` with a capital C already exists. This manifest will need to be put under the existing folder